skdata/mnist/dataset.py

```diff
@@ -49,6 +49,7 @@
     test_labels='4.5K',
     )
 
+
 def _read_int32(f):
```

Comment:
Please change this also in fetch function.
`downloader = urllib.urlopen(url)`
`downloader = urllib.request.urlopen(url)`